docs-pipeline.md

```diff
@@ -0,0 +1,169 @@
+# How content changes in enki
+
+This describes the overall steps that content goes through.
+Several of these can be clumped together instead of being separate and would reduce the amount of validation & debugging/testing.
```

Comment:
This line needs some polish or a rework.
- What are "these?"
- Are they clumped together in the document to form a conceptual block? 
- Are they clumped together process-wise? 
- Are they currently clumped together at all or is it a suggestion that they could be clumped together (and if so is a reference doc the right place for that?)

Reply:
Thanks! I clarified the idea a little bit